SSM/GameManagers/EventManager.java

```diff
@@ -44,12 +46,29 @@ public void onPlayerInteract(PlayerInteractEvent e) {
         }
     }
 
+    // left click in adventure mode for skeleton + zombie
+    // the reason why it randomly doesn't work sometimes
+    @EventHandler
+    public void onPlayerAnimation(PlayerAnimationEvent e) {
+        Player player = e.getPlayer();
+        Ability ability = KitManager.getCurrentAbility(player);
+
+        if (ability == null) {
+            return;
+        }
+
+        if (ability instanceof OwnerAnimationEvent && (e.getAnimationType() == PlayerAnimationType.ARM_SWING)) {
```

Comment:
Are you absolutely certain this won't trigger off of people right clicking fences with a bow in hand?